vocab/templates/vocab.html

```diff
@@ -73,32 +73,57 @@ <h1>Words</h1>
          <br />
          <div class="row">
             <div class="col-xs-6 text-center">
-               <form id="entry" method="POST" action="/_check">
+		    <form id="entry"> 
                   <label for="attempt">Word:</label>
                   <input id="attempt" name="attempt" type="text" width="15"
                      autocomplete="off"  autofocus />
-                  <input type="submit" name="try" />
                </form>
             </div>
          </div>
          <br />
-         {% if session.matches|length is not equalto 0  %}
+     
```

Comment:
```suggestion
     <p id="error"> </p>
```

Spot to put error messages so the HTML displays when someone has inputted an incorrect word.